src/interop/js-interop/js-types.md

```diff
@@ -0,0 +1,189 @@
+---
+title: JS Types
+description: Usage information about the core types in JS interop.
+---
+
+## Motivation
+
+Dart values and JS values belong to separate domains. When compiling to Wasm,
+they belong in separate *runtimes* as well. As such, users should treat JS
+values as foreign types. In order to provide types for JS values, we use the set
+of JS types available in `dart:js_interop`. These are extension types prefixed
+with `JS` and are called "JS types".
```

Comment:
I largely agree with you on the first comment, although I like "To provide *Dart* types for JS values" better.

> similar to "you" vs "the user" in my previous comment, we usually default saying "Dart" or the specific actor (compiler, the package, etc.) instead of "we", or just attributing the action to the user if it's something "we" decided, but they actually do

Done (here and elsewhere)!.

I also like run on sentences (although I don't think qualifies :)), so done!

Reply:
LGTM!